### ISSUE TITLE
All CI tests fail during building HTML documentation with Sphinx

### ISSUE THREAD

**Issue by jesper-friis** [SINTEF/dlite#666]
All CI tests fails with the following message

```
Building HTML documentation with Sphinx

Extension error (autoapi.extension):
Handler <function run_autoapi at 0x7fa80bbbae50> for event 'builder-inited' threw an exception (exception: 'Module' object has no attribute 'doc')
make[2]: *** [doc/CMakeFiles/Sphinx.dir/build.make:75: doc/html/index.html] Error 2
make[1]: *** [CMakeFiles/Makefile2:3252: doc/CMakeFiles/Sphinx.dir/all] Error 2
make: *** [Makefile:166: all] Error 2
Error: Process completed with exit code 2.
Upload artifact
```

See e.g. https://github.com/SINTEF/dlite/actions/runs/6485413846/job/17611432913?pr=648

**Comment by jesper-friis** [comment]
The excact same CI workflows doesn't fail when run through gdb. See https://github.com/EMMC-ASBL/oteapi-dlite/pull/183

**Comment by jesper-friis** [comment]
Temporarily fixed in PR #670
